lib/web_ui/dev/build.dart

```diff
@@ -31,12 +36,20 @@ class BuildCommand extends Command<bool> {
 
   bool get isWatchMode => argResults['watch'];
 
+  int get gomaWorkers {
+    final String gomaWorkersArg = argResults['goma'];
+    if (gomaWorkersArg != null) {
+      return int.parse(gomaWorkersArg);
```

Comment:
This will throw if I pass `-jk`, use tryParse instead?

Reply:
I want it to throw if you pass a non-integer.

Reply:
Or do you think it's better to print a warning, and continue building without goma workers?